browser/browser.go

```diff
@@ -0,0 +1,77 @@
+package browser
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+func Browser(path string) string {
```

Comment:
just a suggestion, I believe the function name is somehow confusing, `Browser` implies creating an object or something, I would just name it `Browse` or `Open` maybe, a name of an action instead of the current naming.